CHANGELOG.md

```diff
@@ -3,6 +3,14 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [vector-0.10.1] - 2022-04-22
+
+### Vector
+
+#### Features
```

Comment:
Shouldn't this be "Bug Fixes" 🤔

Reply:
I rebased and force pushed to rename commit to `fix(vector)`.

Reply:
Yeah, you right 😵‍💫